handler/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "ezbids-handler",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "dev": "ts-node-dev --respawn --transpile-only handler.ts",
+    "prod": "tsc && node ./build/handler.js"
```

Comment:
are files transpiled into JS when built? Just double checking that the prod script here was intentional and shouldn't instead be: "prod": "tsc && node ./build/handler.ts"